### ISSUE TITLE
values-es already exists

### ISSUE THREAD

**Issue by danielehrhardt** [mjwheatley/cordova-plugin-android-fingerprint-auth#68]
Hi,
i get the following error when i want to install this plugin to a ionic 2 app:

![image](https://cloud.githubusercontent.com/assets/19570043/24587456/5d433626-17b7-11e7-867f-39c4edfe7320.png)

```
Failed to install 'cordova-plugin-android-fingerprint-auth':CordovaError: Uh oh!
"E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\res\values-es" already exists!
    at copyNewFile (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\lib\pluginHandlers.js:248:15)
    at install (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\lib\pluginHandlers.js:43:17)
    at Object.process (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\ActionStack.js:56:25)
    at PluginManager.doOperation (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:117:20)
    at PluginManager.addPlugin (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:147:17)
    at E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\Api.js:254:18
    at _fulfilled (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:796:13)
    at E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:556:49

Failed to restore plugin "cordova-plugin-android-fingerprint-auth" from config.xml. You might need to try adding it again. Error: Uh oh!
"E:\BussmannIT\Projekte\Netz-App\netz-app\platforms\android\res\values-es" already exists!
```

Thank you for Help

**Comment by mjwheatley** [comment]
What have you tried so far?

Do you already have a values-es folder with other Spanish translations?

**Comment by danielehrhardt** [comment]
i have this Problem with this Plugin
https://github.com/wymsee/cordova-imagePicker

**Comment by mjwheatley** [comment]
Maybe try using --force

cordova plugin add --force

Introduced in version 6.1. Forces copying source files from the plugin even if the same file already exists in the target directory.

**Comment by danielehrhardt** [comment]
no it is not working
```
Error during processing of action! Attempting to revert...
        Failed to install 'cordova-plugin-android-fingerprint-auth': Error
        at copyNewFile
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\lib\pluginHandlers.js:248:15)
        at install
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\lib\pluginHandlers.js:43:17)
        at ActionStack.process
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\ActionStack.js:56:25)
        at PluginManager.doOperation
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:117:20)
        at PluginManager.addPlugin
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:147:17)
        at C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\Api.js:254:18
        at _fulfilled
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:854:54)
        at self.promiseDispatch.done
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:883:30)
        at Promise.promise.promiseDispatch
        (C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:816:13)
        at C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\cordova\node_modules\q\q.js:570:49
        Error: Uh oh!
        "C:\Users\Daniel\Desktop\Alles\Projekte\Netz-App\netz-app\platforms\android\res\values-es" already exists!

```

**Comment by mjwheatley** [comment]
Most likely this is not an issue with the plugin, but with your project setup. Reference issue#84.

I asked this question previously and received no answer:
Do you already have a values-es folder with other Spanish translations?

What framework are you using to compile your project (i.e. is it a Cordova project, Ionic, Meteor, etc...)

**Comment by mjwheatley** [comment]
Also, please look at this resolved issue as well #61.
